editor/src/clj/editor/properties_view.clj

```diff
@@ -223,12 +224,15 @@
   (.consume event)
   (let [target (.getTarget event)
         property (property-fn)
-        {:keys [key node-ids edit-type]} (property-fn)
+        {:keys [key node-ids edit-type]} property
         {:keys [precision from-type to-type]} edit-type
-        [x y] [(.getX event) (.getY event)]
+        x (.getX event)
+        y (.getY event)
         [prev-x prev-y] (ui/user-data target ::position)
         delta-x (- x prev-x)
         delta-y (- prev-y y)
+        script-property? (partial g/node-instance? script/ScriptPropertyNode)
+        key-fn (fn [node-id] (if (script-property? node-id) :value key))
```

Comment:
Script properties are represented by dedicated graph nodes as far as I understand. This should fix the issue, but it looks like `properties/set-values!` abstracts this away. Not sure if its worth it changing the implementation to use that instead.

Reply:
Hmm... I think we should use `properties/set-values!` or implement some function there specifically for drag operations. There is a set of rules for changing properties, and the rules should preferably be located in some shared place (like `properties` ns) to separate concerns. I don't quite remember this set of rules, but it seems that these property maps may include `:prop-kw` keyword to denote the right key to use for setting the value. The logic should probably check for that instead of checking for the node instance: script properties define the `prop-kw` here: https://github.com/defold/defold/blob/0272aeb58dd014dbe30a0adcc0dfecc52e00bfb7/editor/src/clj/editor/code/script.clj#L187